packages/diagrams/backend/sirius-components-diagrams/src/main/java/org/eclipse/sirius/components/diagrams/LayoutDirection.java

```diff
@@ -0,0 +1,22 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.components.diagrams;
+
+/**
+ * The layout direction used by the {@link ListLayoutStrategy} to position the children of a node.
+ *
+ * @author gcoutable
+ */
+public enum LayoutDirection {
+    COLUMN
+}
```

Comment:
The ADR mentions that the `row` direction should be supported, and if we only support a single direction for the time being, we should not need this.